hibernate-search-plugins/org.jboss.tools.hibernate.search/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,30 @@
+Manifest-Version: 1.0

```

Comment:
I mean that I would expect this file to be automatically generated during the build. No?

In that case it should not be committed into source control.

Maybe it's not generated. I simply expect it would be generated as that would make it less error prone, for example to have the right versions declared in here when you change the versions in the pom file.